src/services/LedgerService.ts

```diff
@@ -20,6 +20,34 @@ export default class LedgerService extends WalletService {
       ? bitcoin.networks.bitcoin : bitcoin.networks.testnet;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  public reconnect(): Promise<void> {
+    return new Promise<void>((resolve, reject) => {
+      try {
+        LedgerTransportService.newInstance();
```

Comment:
how can we guarantee the ledger is connected now?
Should we call isConnected from here after we get a newer instance?

Reply:
To avoid a loop, we will verify if is connected, if is not, we will try to reconnect once, and then, try to obtain the wallet's data, if we can't connect, the app will raise an exception and the application will show a popup to the user.